frontend/src/components/dashboard/dashboard.tsx

```diff
@@ -1,5 +1,32 @@
-import { FC } from 'react';
+import { FC, useEffect } from 'react';
```

Comment:
```suggestion
import { FC } from 'react';
```
import from hooks folder